Core/Utilities/RabbitMQAdmin.py

```diff
@@ -0,0 +1,138 @@
+"""RabbitMQAdmin module serves for the management of the internal RabbitMQ
+   users database. It uses rabbitmqctl command. Only the user with the right
+   permissions can execute those commands.
+"""
+import re
+from DIRAC import S_OK, DError
```

Comment:
Fixed in the next push.
Btw his module  should be in the second PR so I will move it there.